ui/src/components/pages/Pipeline/partials/Graph/partials/NodeInfo/partials/Pods/partials/PodDetails/partials/Metrics/partials/common/FiltersDropdown/index.tsx

```diff
@@ -29,6 +29,13 @@ export interface FiltersDropdownProps {
   setFilters: any;
 }
 
+const periodData = [
```

Comment:
periods of "1m", "5m" etc are defined at the backend?

Reply:
found it https://github.com/numaproj/numaflow/blob/11c0d2b40c314d81517fe8afeea30edd50f3cda4/rust/numaflow-core/src/metrics.rs#L841